package.json

```diff
@@ -9,7 +9,7 @@
     "@webassemblyjs/helper-module-context": "1.9.0",
     "@webassemblyjs/wasm-edit": "1.9.0",
     "@webassemblyjs/wasm-parser": "1.9.0",
-    "acorn": "^6.2.1",
+    "acorn": "^7.0.0",
```

Comment:
The [advisory](https://www.npmjs.com/advisories/1488) says to upgrade to `7.1.1`, `7.0.0` still has the vulnerability.

Reply:
this constraint `^7.0.0` will install latest version which is 7.1.1

Reply:
Yes, but we don't ever want to be using a version with the vulnerability, so the constraint should be:

```suggestion
    "acorn": "^7.1.1",
```

This ensures that package managers can never accept a lesser version even if one is available by another package (i.e to deduplicate) :)